app/src/main/java/nerd/tuxmobil/fahrplan/congress/changes/SessionChangeParametersFactory.kt

```diff
@@ -81,47 +81,47 @@ class SessionChangeParametersFactory(
             title = SessionChangeProperty(
                 value = title,
                 contentDescription = title,
-                changeState = changeStateOf(session, session.changedTitle),
+                changeType = changeStateOf(session, session.changedTitle),
```

Comment:
You might also want to change the name of the function `changeStateOf` to `changeTypeOf`.